.github/workflows/outdated-dependencies.yml

```diff
@@ -1,37 +1,43 @@
 name: Check outdated dependencies
 on:
-  # TODO: uncomment schedule
-  # schedule:
-  #   # Runs Mon-Fri at 0:00 UTC ( 00:00 EST )
-  #   - cron: "0 0 * * 1-5"
+  schedule:
+    # Runs Mon-Fri at 0:00 UTC ( 00:00 EST )
+    - cron: "0 0 * * 1-5"
```

Comment:
Once every two weeks would be more appropriate. We will never be 100% up to date with the same major version of everything, often up to date modules require older but maintained versions of other modules, etc. If we generate this report daily we will become blind to it (:

Reply:
Would every week do?
I found a syntax to run it every 2 week: `0 8 * * MON test $((10#$(date +\%W)\%2)) -eq 1`, but it's not compatible with GA